src/components/timeline/index.css

```diff
@@ -28,24 +28,59 @@
   list-style: none;
 }
 
-.timelineToggle {
-  position: absolute;
-  top: 0;
-  left: -150px;
+.timelineSettings {
+  display: flex;
+  height: 25px;
+  flex-flow: row nowrap;
+  padding: 0;
+  border-bottom: 1px solid var(--grey-30);
+  color: var(--ink-70);
+  line-height: 25px;
+  white-space: nowrap;
+}
+
+.timelineSettingsToggle {
   display: flex;
-  width: 150px;
-  height: 22px;
-  font-size: 9px;
+  padding: 0 8px;
+  border-right: 1px solid var(--grey-30);
 }
 
-.timelineToggleInput {
-  width: 12px;
-  height: 12px;
-  margin: 0 3px;
+.timelineSettingsToggleInput {
+  margin: 0 5px;
   vertical-align: middle;
 }
 
-.timelineToggleLabel {
+.timelineSettingsToggleLabel {
   display: flex;
   align-items: center;
 }
+
+.timelineSettingsHiddenTracks {
```

Comment:
nit: maybe name this `.timelineSettingsHiddenTracksButton` to make it more obvious that we're styling the button?

Reply:
also we'll have to implement the hover and active style... I'd prefer that you implement this in photon-components so that it's easily reusable, but I wouldn't block for it.

See https://design.firefox.com/photon/components/buttons.html for the spec

Reply:
I'd like to keep the name, as it's using the name concatenation pattern. I added a comment:

`/* Make sure and override all rules that have to do with button defaults. */`